src/main/java/frc/robot/subsystems/swerve/states/aimassist/AimAssist.java

```diff
@@ -0,0 +1,7 @@
+package frc.robot.subsystems.swerve.states.aimassist;
+
+public enum AimAssist {
+
+	NONE;
+
+}
```

Comment:
maybe add in template accesspotiton something to add stuff here if needed

Reply:
in another pr